File: E-mart_.Net/Backend/Dtos/OrderDTO.cs
namespace E_mart.Dtos
{
    public class OrderDTO
    {
        public int OrderId { get; set; }
        public double Amount { get; set; }
        public DateTime? OrderDate { get; set; }
        public string? PaymentMethod { get; set; }
        public string? Status { get; set; }
        public int CartId { get; set; }
        public int UserId { get; set; }
    }
}

File: E-mart_java/Backend/src/main/java/com/example/Dtos/OrderDto.java
package com.example.Dtos;

import java.util.List;
import com.example.Models.CartItem;

public class OrderDto {
    private int userId;
    private int cartId;
    private double amount;
    private String paymentMethod;
    private List<CartItem> cartItems; // Ensure CartItem is a valid model

    // Getters and setters
    public int getUserId() {
        return userId;
    }
    public void setUserId(int userId) {
        this.userId = userId;
    }
    public int getCartId() {
        return cartId;
    }
    public void setCartId(int cartId) {
        this.cartId = cartId;
    }
    public double getAmount() {
        return amount;
    }
    public void setAmount(double amount) {
        this.amount = amount;
    }
    public String getPaymentMethod() {
        return paymentMethod;
    }
    public void setPaymentMethod(String paymentMethod) {
        this.paymentMethod = paymentMethod;
    }
    public List<CartItem> getCartItems() {
        return cartItems;
    }
    public void setCartItems(List<CartItem> cartItems) {
        this.cartItems = cartItems;
    }
}
